music_assistant/controllers/media/artists.py

```diff
@@ -280,6 +288,7 @@ async def _add_library_item(self, item: Artist | ItemMapping) -> int:
         if item.mbid == VARIOUS_ARTISTS_MBID:
             item.name = VARIOUS_ARTISTS_NAME
         # no existing item matched: insert item
+        assert self.mass.music.database is not None  # for type checking
```

Comment:
instead of doing 99999 asserts for a value that will always be there, we should consider making mass.music.database a property with the type check in there and the actual db conn as private variable (which we return in the property)